src/site/content/en/blog/2022-state-of-css/index.md

```diff
@@ -0,0 +1,1902 @@
+    alt="Screenshot of the Bungee Spice font with the word DUNE.", 
+    width="564", 
+    height="169" 
+  %}
+  <figcaption>
+    Bungee Spice font shown with custom colors, source from <a href="https://developer.chrome.com/blog/colrv1-fonts/">https://developer.chrome.com/blog/colrv1-fonts/</a>
+  </figcaption>
+</figure>
+
+With more and more variable fonts and color fonts becoming available, web
+typography is on a very magnificent path towards rich customization and creative
+expression.
+
+**Resources**  
+Github: [https://github.com/googlefonts/colr-gradients-spec](https://github.com/googlefonts/colr-gradients-spec)  
+Chrome Developers: [https://developer.chrome.com/blog/colrv1-fonts/](https://developer.chrome.com/blog/colrv1-fonts/)  
+BlinkOn developer explainer video: [https://www.youtube.com/watch?v=BmqYm5Wwz8M](https://www.youtube.com/watch?v=BmqYm5Wwz8M)  
+
+### Viewport units
+
+{% BrowserCompat 'css.types.length.dvh' %}
+
+{% Img 
+  src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/jE6JZPgLnHO8HQWtfgNE.png", 
+  alt="Graphic showing how the device screen and the browser window and an iframe, all have different viewports.", 
+  width="800", 
+  height="501" 
+%}
+
+Before the new viewport variants, the web offered physical units to assist in
+fitting viewports. There was one for height, width, smallest size (vmin), and
+largest side (vmax). These worked well for many things, but mobile browsers
+introduced a complexity. 
+
+On mobile, when loading a page, the status bar with the url is shown, and this
+bar consumes some of the viewport space. After a few seconds and some
+interactivity, the status bar may slide away to allow a bigger viewport
+experience for the user. But when that bar slides out, the viewport height has
+changed, and any `vh` units would shift and resize as their target size changed.
+In later years, the `vh` unit specifically needed to decide which of the two
+viewport sizes it was going to use, because it was causing jarring visual layout
+issues on mobile devices. It was determined that the `vh` would always represent
+the largest viewport.
+
+```css
+.original-viewport-units {
+  height: 100vh;
+  width: 100vw;
+  --size: 100vmin;
+  --size: 100vmax;
+}
+```
+
+After the new viewport variants, small, large, and dynamic viewport units are
+made available, with the addition of [logical
+equivalents](/learn/css/logical-properties/) to the physical ones. The idea is
+to give developers and designers the ability to choose which unit they want to
+use for their given scenario. Maybe it's ok to have a small jarring layout shift
+when the status bar goes away, so then `dvh` (dynamic viewport height) could be
+used without worry.
+
+{% Img 
+  src="image/vS06HQ1YTsbMKSFTIPl2iogUQP73/KtZ9bXwtzmo7hguwQOWO.png", 
+  alt="ALT_TEXT_HERE", 
+  width="800", 
+  height="664" 
+%}
+
+Here's a complete list of all the new viewport unit options made available with
+the new viewport variants:
+
+<div class="table-wrapper scrollbar">
```

Comment:
not sure that a table is best for this, if you use one however you can't use markdown inside it.

Reply:
fixed 👍🏻